packages/transforms/mocking/yaml-config.graphql

```diff
@@ -0,0 +1,44 @@
+extend type Transform {
+    """
+    Mocking configuration for your source
+    """
+    mocking: MockingConfig
+}
+
+type MockingConfig {
+    """
+    If this expression is truthy, mocking would be enabled
+    """
+    if: Boolean
+    """
+    Do not mock any other resolvers other than defined in `mocks`
+    """
+    preserveResolvers: Boolean
+    """
+    Mock configurations
+    """
+    mocks: [MockingFieldConfig]
```

Comment:
This should be required, right?

Reply:
Not really. graphql-tools has its own mocks for basic types such as `Hello World` for string, random UUID for `ID` etc...